galleries/users_explain/axes/tight_layout_guide.py

```diff
@@ -9,15 +9,17 @@
 
 How to use tight-layout to fit plots within your figure cleanly.
 
+.. tip::
+
+   *tight_layout* was the first layout engine in Matplotlib. The more modern
+   and more capable :ref:`constrained_layout <constrainedlayout_guide>` should
```

Comment:
```suggestion
   and more capable :ref:`Constrained Layout <constrainedlayout_guide>` should
```
To be consistent with #25144, which removed the underscore in the constrained layout guide.